### PR TITLE
Pythonify

### PR DESCRIPTION
**Pull request by zconnelly13**
@0bmxa This PR is based off of #1  -- to see just the difference between the two use this link https://github.com/0bmxa/why-no-network/compare/5de9f00...pythonify

This PR basically just changes `camelCase` to `snake_case` it also changes `if len(some_list) == 0:` to `if not some_list`

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -8,70 +8,75 @@
...
+    # if google_dns_ping != 'Down':
     #     return
 
     # NOTE: algo5 does not respond to pings
     # vpnHostPing = ping('138.197.186.179')
     # print('VPN ping:     %s' % vpnHostPing)
 
+    print('')
+    print('Trying to find captive portal... [ALPHA]')
+    print('')
```
Comment (0bmxa) [inline main.py]: What's the benefit of this? Also this doesn't replace `/r`

**Review by 0bmxa — CHANGES_REQUESTED** [review]
Looks more python 👍, but I find the empty prints irritating

File: main.py (hunk 1 above)
Comment (zconnelly13) [reply main.py]: Just a matter of preference I guess -- changed it back.

Also TIL --> https://stackoverflow.com/questions/1761051/difference-between-n-and-r

```
The only reason to use \r is if you're writing to a character terminal 
(or more likely a "console window" emulating it) and want the next line you write 
to overwrite the last one you just wrote (sometimes used for goofy "ascii animation" 
effects of e.g. progress bars)
```

**Comment by zconnelly13** [comment]
Wow the merge conflicts with myself, hmmm what did I fuck up here lol

**Review by 0bmxa — APPROVED** [review]
LGTM

**Pull request merged**